src/NodaTime/ZonedDateTime.cs

```diff
@@ -628,6 +628,20 @@ public ZonedDateTime WithCalendar([NotNull] CalendarSystem calendar)
         [Pure]
         public OffsetDateTime ToOffsetDateTime() => offsetDateTime;
 
+        /// <summary>
+        /// Deconstruct this <see cref="ZonedDateTime"/> into its components.
+        /// </summary>
+        /// <param name="localDateTime">The <see cref="LocalDateTime"/> component.</param>
+        /// <param name="dateTimeZone">The <see cref="DateTimeZone"/> component.</param>
+        /// <param name="offset">The <see cref="Offset"/> component.</param>
+        [Pure]
+        public void Deconstruct(out LocalDateTime localDateTime, [NotNull]out DateTimeZone dateTimeZone, out Offset offset)
```

Comment:
Mega-nit: could you add a space after the attribute? So `[NotNull] out DateTimeZone dateTimeZone`